platform/broadcom/sonic-platform-modules-inventec/d6356/utils/inventec_d6356_util.py

```diff
@@ -141,7 +141,7 @@ def system_install():
     status, output = exec_cmd("rmmod lpc_ich ", 1)
```

Comment:
should use blacklist module as the standard way to block loading certain drivers.

Reply:
This part is modified after previous feedback, please help to review, thank you!